### PR TITLE
Note the difference in performance between the KV get types

### PR DESCRIPTION
**Pull request by a-robinson**
@steveklabnik

### CODE REVIEW

**Comment by a-robinson** [comment]
@signalnerve

**Comment by signalnerve** [comment]
fascinating!

File: content/reference/apis/kv.md [hunk 1]
```diff
@@ -153,6 +153,10 @@ also specified which will convert a JSON value into an object before
 returning it to you. For large values you can request a `ReadableStream`, and
 for binary values an `ArrayBuffer`.
 
+For large values, the choice of `type` can have a noticeable effect on latency
+and CPU usage. For reference, the `type`s can ordered from fastest to slowest
```
Comment (harrishancock) [inline content/reference/apis/kv.md]: /can ordered/can be ordered/

Comment (a-robinson) [reply content/reference/apis/kv.md]: Whoops, thanks. I'll send a fix.

**Pull request merged**